main.js

```diff
@@ -0,0 +1,7 @@
+var tableElement = document.getElementsByClassName("table");
```

Comment:
Don't use library (Bootstrap in this case) class names to target elements with your custom scripts. Use your own custom class names (ex: `grade-table`) instead.